py/picca/data.py

```diff
@@ -1052,7 +1053,14 @@ def from_fitsio(cls, hdu, pk1d_type=False):
             delta_name = "DELTA"
 
         delta = hdu[delta_name][:].astype(float)
-        log_lambda = hdu['LOGLAM'][:].astype(float)
+
```

Comment:
ok, this is just dealing with the new output files and allows putting both binnings in. So all downstream routines can basically work the same way as they still get log-lambda variables. Probably would be good to store the type of binning used in a class variable as well for tracking this downstream (e.g. when computing P1d we'd want to know later which binning this is in, while one could check this explicitely it would be simpler to have this accessible from here...)

Reply:
Yes, that is correct. Going downstream is the part of the code that we still need to refactor, but this is another huge project that we will need to find someone to do. There is a lot of repeated code between `cf.py` and `xcf.py` and some issues when computing the metal matrixes with different binning. I guess we should add this tracking requirement to the list of pending issues there

Reply:
yes, makes sense, restructuring everything is a long-term task. But it's good to know that everything can at least still read the data. Is evenly spaced binning in LOGLAM implicitely assumed when running (x)cf? For P1d it is, but I know the changes needed from my other branch, need to port this rather sooner than later